python/GangaDirac/Lib/Files/uploadScript.py

```diff
@@ -1,39 +1,40 @@
 
 import os
 ## NB parseCommandLine first then import Dirac!!
+import datetime
+import gzip
+import sys
+from contextlib import closing
 from DIRAC.Core.Base.Script import parseCommandLine
-parseCommandLine()
 from DIRAC.Interfaces.API.Dirac import Dirac
+parseCommandLine()
 dirac=Dirac()
 errmsg=''
 file_name='###UPLOAD_FILE###'
 file_label=file_name
 
 compressed = ###COMPRESSED###
 if compressed:
-    import gzip
     file_name += '.gz'
     f_in = open(file_label, 'rb')
     f_out = gzip.open(file_name, 'wb')
     f_out.writelines(f_in)
     f_out.close()
     f_in.close()
 
-lfn= os.path.join('###LFN_BASE###', file_name)
+lfn=os.path.join('###LFN_BASE###', file_name)
 wildcard='###WILDCARD###'
 storage_elements=###SEs###
 
-from contextlib import closing
 with closing(open('###LOCATIONSFILE_NAME###','ab')) as locationsfile:
     for se in storage_elements:
+        sys.stdout.write('\nUploading file: \"%s\" as \"%s\" at \"%s\"\n' % (file_name, lfn, se))
```

Comment:
I was wanting to be certain these were escaped, also I was originally wrapping the whole string in `"` rather than `'`.